### ISSUE TITLE
View max value for metrics when comparing vs hyper-parameters 

### ISSUE THREAD

**Issue by oren-oneview** [allegroai/trains#131]
Currently comparing performance metrics between experiments against their hyperparams, only allows to use the last calculated performance metrics. 
(i.e. selecting experiments->compare->Hyper Parameters: Performance Metric).
![image](https://user-images.githubusercontent.com/55651296/81476062-69325c80-9218-11ea-8a07-2c53b45032e6.png)

When viewing the experiments not in compare mode, i.e. the table view, we can also view the max value for the calculated performance metrics. 
![image](https://user-images.githubusercontent.com/55651296/81476144-de059680-9218-11ea-82d3-089948e9ced8.png)

It would be great to see also the maximum value in the comparison via hyper-parameters view.

**Comment by oren-oneview** [comment]
Sorry, just saw it is possible by changing to max-values ( select scalars->maxValues).

**Comment by bmartinn** [comment]
Hi @oren-oneview ,
You are most definitely correct :) 
I'll update here once we have it, hopefully in the next *trains-server* release (due in a few weeks)

**Comment by bmartinn** [comment]
Hi @oren-oneview 
Good news :tada: the v0.15 is out with this feature added :)
you can already check it on the [demo-server](https://demoapp.trains.allegro.ai)
